fstat/controller.py

```diff
@@ -1,50 +1,87 @@
 from collections import Counter
-from flask import render_template, redirect, url_for
+from flask import render_template, redirect, url_for, request
 from fstat import app, db
-from fstat.lib import x_days_ago
 from model import Failure, FailureInstance
-from datetime import datetime
+from datetime import datetime, timedelta
 
 
 @app.route("/")
 def index():
-    return redirect(url_for('weekly_overall_summary', num=1))
+    return redirect(url_for('overall_summary'))
 
 
-@app.route("/weeks/<int:num>")
-def weekly_overall_summary(num=None):
-    if num > 12:
-        num = 12
-    cut_off_date = (x_days_ago(num*7))
-    failure_instances = FailureInstance.query.filter(
-            FailureInstance.timestamp > cut_off_date)
+@app.route('/summary')
+def overall_summary():
+    ''' 
+        Shows overall summary
+        Params
+            - start_day: date with format yy-mm-dd, if start date is none it is defaulted to the last monday
+            - end_day: date with format yy-mmd-dd, if end date is none it is defaulted to today
+    '''
+    start_date = request.args.get('start_date')
```

Comment:
I wanted to do that before making PR but I was stuck in choosing a meaningful name for the function.